WoD/TheEverbloom/Yalnu.lua

```diff
@@ -130,9 +130,9 @@ function mod:ColossalBlow(args)
 	self:PlaySound(args.spellId, "alarm")
 	colossalBlowCount = colossalBlowCount + 1
 	if colossalBlowCount % 3 ~= 1 then -- 2, 3, 5, 6...
-		self:CDBar(args.spellId, 15.8)
+		self:CDBar(args.spellId, 15.8, CL.count:format(args.spellName, colossalBlowCount))
```

Comment:
Should the count also be included in the message?

Reply:
hmm that's for you to decide, imo people are just gonna look at timers to prep defensives and might be too little time to react to message

Reply:
Usually I tend to keep the bar and message consistent, so I'd lean towards adding it to the message as well.